pkg/helm/install_chart.go

```diff
@@ -3,11 +3,23 @@ package helm
 import (
 	"helm.sh/helm/v3/pkg/action"
 	"helm.sh/helm/v3/pkg/chart/loader"
+	"helm.sh/helm/v3/pkg/cli"
+	"helm.sh/helm/v3/pkg/cli/values"
+	"helm.sh/helm/v3/pkg/getter"
 )
 
-func InstallChart(ns, name, url string, conf *action.Configuration) (interface{}, error) {
+func InstallChart(ns, name, url string, vals *values.Options, conf *action.Configuration) (interface{}, error) {
```

Comment:
Since this is a service, we should not retrieve values for template engine from env variables, extra files, etc. We should get them through service request. Hence, `vals` should be declared as 
```
vals map[string]interface{})
``` 

and we can skip the block added at line 14.

Reply:
passing val as `map[string]interface{}`